app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventDetails/ui/EventDetailsViewModel.kt

```diff
@@ -266,7 +268,22 @@ class EventDetailsViewModel(
         val calendar = Calendar.getInstance()
         calendar[year, month, day, 0, 0] = 0
         calendar[Calendar.MILLISECOND] = 0
+
```

Comment:
This parsing is outside the form module so it depends on the implementation, maybe we could think in a way that at least the form module contains this